src/store/notificationContext.js

```diff
@@ -0,0 +1,25 @@
+import * as signalR from "@microsoft/signalr";
```

Comment:
Do not import everything from the package. Import only what you need. In this case `HubConnectionBuilder`.